### ISSUE TITLE
Nehemoth never spawns

### ISSUE THREAD

**Issue by melvinczyk** [lender544/FromTheShadows-1.20.1#1]
With every version of the mod, nehemoth never spawns in general. Even with the custom spawns in the config turned all the way up, it never spawns in fortresses, underground caves, or anywhere at all.

**Comment by Jimmy-Macandcheese** [comment]
I've had the same issue with nehemoth not spawning.

**Comment by KRO5I5** [comment]
Same here, I used SpawnBalancedUltility to increase it to 80 weight to spawn in the (the_end). Nothing shows up. Even when im in caves.

**Comment by KRO5I5** [comment]
> > Same here, I used SpawnBalancedUltility to increase it to 80 weight to spawn in the (the_end). Nothing shows up. Even when im in caves.
> 
> Yeah I did the exact same thing, the only way I could get them to spawn was with InControl but it was janky. I ended up not using this version of the mod but Sonmonk's original version because I still had the jar from another modpack.

By chance is that version 1.20.1 for forge. If so may I get the link to replace the file until this issue is fixed.

**Comment by melvinczyk** [comment]
@KRO5I5 Yeah the version is for 1.20.1 forge but I am not sure I am allowed to upload the file anywhere because of the author took the mod down. But if you make an issue on any of my repos I can send the file to you.

**Comment by melvinczyk** [comment]
@KRO5I5 There is no link to the file because the author took the mod down. But it is for 1.20.1 forge but I don't think I can upload it anywhere because of licensing but I could send it to you directly.

**Comment by KRO5I5** [comment]
> @KRO5I5 There is no link to the file because the author took the mod down. But it is for 1.20.1 forge but I don't think I can upload it anywhere because of licensing but I could send it to you directly.

Sure, here is my email, reysilv39@gmail.com

**Comment by komdreupau** [comment]
Dude can I also please get this .jar? komdreupau@outlook.com I didn't realize that it didn't spawn yet, I'm still building my modpack. I came here because I can't seem to damage the statue form, even after fixing the loot table coal drop negative value.
